<!-- Launch the Autonomy module, which is a single node. -->

<launch>
  <arg name="plan" default="Demo.plx"/>
  <node pkg="ow_autonomy"
        name="autonomy_node"
        type="autonomy_node"
        args="$(arg plan)"
        output="screen" />
  <node pkg="ow_autonomy"
        name="GuardedMoveServer"
        type="GuardedMoveServer"
        output="screen" />
  <node pkg="ow_autonomy"
        name="planner_node"
        type="planner_node"
        output="screen" />
</launch>
